.travis.yml

```diff
@@ -51,7 +51,7 @@ before_script:
   - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d && echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - composer selfupdate
   - composer config -q -g github-oauth.github.com $GITHUB_OAUTH_TOKEN
-  - composer global require phpunit/phpunit fabpot/php-cs-fixer --no-update
+  - composer global require "phpunit/phpunit:^4.0" fabpot/php-cs-fixer --no-update
```

Comment:
* No double quote
* `^4.8`

Will be easier to merge on master (modification already done).